### ISSUE TITLE
Review accessibility style guidance for fax numbers

### ISSUE THREAD

**Issue by megzehn** [department-of-veterans-affairs/va.gov-team#77381]
## Issue Description 
As someone reading content on VA.gov, I don't want to accidentally call a fax number. 
As a writer on VA.gov, I need to know how to style fax numbers in content.

Open questions:
Do we need to wrap code around fax numbers to prevent them from getting linked automatically? Or is it sufficient to just make it clear in our text that the number is a fax number. 

Background: [This question and issue came out of previous aria label research.](https://github.com/orgs/department-of-veterans-affairs/projects/929/views/11?filterQuery=assignee%3Amegzehn+aria&pane=issue&itemId=30706290)

Example:
https://www.va.gov/decision-reviews/board-appeal/

---
## Tasks
- [ ] _a11y reviews 
- [ ] content adds documentation to style guide

## Acceptance Criteria
- [ ] _We have documented guidance in our style guide
---

**Comment by EvanAtCoforma** [comment]
# Exploring A11y Options

## Option 1: Wrap code to deactivate link

Using code to deactivate the link would ensure the user would not be able to access it as they cycle through the links on the page. There's a good chance an average screen reader user will navigate via headings/heading list or scan the page using the arrow keys so they'll be able to reach the "Option 4: by fax" heading and explore within that section to find the fax number. This may be the safest option for users as it will eliminate the chance of a user activating the link, though varying skill level of SR users would determine how long it would take for them to find the number since it won't gain focus by tabbing through the page. 

Due to the fax number link being deactivated, there must also be some visual distinction to show visual users that the number is important for that section. Making the fax number text bold would be the best option to remain consistent with the rest of the page that highlights important text. 

## Option 2: Clearly state fax number in body text

While stating that the number is a fax number in the body text, the link would still be active, therefore can be accessed by users navigating using the TAB key/Link list. If a user chooses to navigate one of those two ways, the body text will be ignored by the screen reader since that text is not active, while the link is. When navigating to the link, the VoiceOver output says, "link, 844-678-8979," which does not provide any context to the user. If we were to go with this option, we'd need to add programmatic context to the fax number link so a user knows it's not just a regular phone number. So the SR text could read, "link, fax number: 844-678-8979" (example below). As long as there is context added to the link, a user can navigate to it and understand what the link is. 

![image](https://github.com/department-of-veterans-affairs/va.gov-team/assets/129999082/a6b72ca1-0839-4233-aa9d-54aab6766447)

## My A11y opinion

I'd choose option 1, as it eliminates any chance of user error. A beginner user may take longer to find the information, but at least there wouldn't be any possibility of activating it due to it being a link. Not sure if there's a more efficient way to avoid automatic linking, but overall, the fax number should be static text. 


cc: @megzehn , @coforma-terry

**Comment by megzehn** [comment]
Thank you @EvanAtCoforma! This makes sense to me.  

Could you add the code we should use to wrap around the fax number to deactivate the link?

**Comment by EvanAtCoforma** [comment]
Hey @megzehn , not sure how this ticket got buried but clearly I never got back to you about this!

I mentioned there being some code to wrap around it as that was part of the original question, but there's no need to wrap any specific code around it to make it static text. You can just delete the `<va-telephone>` element containing the telephone number and the number to the sentence above. Check out the screenshots below:

Current markup:
![image](https://github.com/department-of-veterans-affairs/va.gov-team/assets/129999082/1cd90f83-6dd2-47ec-b2f3-068815320249)

Recommended markup:
![image](https://github.com/department-of-veterans-affairs/va.gov-team/assets/129999082/1da15644-013a-459e-8daa-a3c1fa4d354a)

Let me know if you have any questions!

**Comment by megzehn** [comment]
Thanks, @EvanAtCoforma. Sorry if I didn't make this clearer in my initial write-up, but the linking attributes are happening automatically after we add the fax number in the CMS. I'm not totally sure at what stage the linking attributes get added (would it be the browser? or our CMS?).

I just confirmed in Drupal that we don't have any `<a href> `or` <va-telephone>` coding attached to the fax number on the [page I referenced](https://www.va.gov/decision-reviews/board-appeal/), but it's still linking for me when I view the live page in Chrome. 


What we have entered in Drupal: 

![Image](https://github.com/department-of-veterans-affairs/va.gov-team/assets/72934749/e62ccd7a-4e0f-4fe4-8391-b456a562aa02)



What's showing on production: 

![Image](https://github.com/department-of-veterans-affairs/va.gov-team/assets/72934749/cce25bb9-516c-4e89-be96-4f321d4151e8)



I should also mention that this isn't an issue that surfaced from Veterans or real-life users. It came from an internal observation while working on ticket #60219.  We may need to back up and try to assess how big of a problem it's causing users.

**Comment by megzehn** [comment]
Thanks Evan!

I chatted with Danielle and added some updated guidance to the style guide to recommend not adding aria labels or links. 

[That guidance lives here. ](https://design.va.gov/content-style-guide/dates-and-numbers#fax-numbers)

**Comment by megzehn** [comment]
Yes, @strelichl! It just needs to be added to the capacity tracker. I'm happy to do that later, but if you want to take care of it, go for it!

**Comment by megzehn** [comment]
Amazing, thank you!
